lib/chef/knife/bootstrap/chef-full.erb

```diff
@@ -19,32 +27,128 @@ exists() {
   fi
 }
 
+# do_wget URL FILENAME
+do_wget() {
+  echo "trying wget..."
+  wget -O "$2" "$1" 2>/tmp/stderr
```

Comment:
I already fixed that in the latest omnitruck install.sh versions.  There's a portable mktemp-like code which is secure enough.